cmd/access/node_builder/unstaked_access_node_builder.go

```diff
@@ -126,6 +126,7 @@ func (anb *UnstakedAccessNodeBuilder) Initialize() error {
 	anb.enqueueConnectWithStakedAN()
 
 	anb.PreInit(anb.initUnstakedLocal())
+	anb.PreInit(cmd.DynamicStartPreInit)
```

Comment:
This is synchronizing startup for newly staked nodes right? This is an unstaked node, so I don't think it's needed here.

Reply:
https://github.com/onflow/flow-go/pull/1893/commits/ce2914b1954138e89a40c9b84d93e56014876c00